### PR TITLE
Provide defaultValue on input types in schema introspection

### PR DESCRIPTION
**Pull request by benzolium**
Resolves https://github.com/movio/bramble/issues/254

### CODE REVIEW

**Review by andyyu2004 — COMMENTED** [review]
Thanks! I'd like to see a few more test cases with more types of the defaultValue (enums, input objects etc).

**Comment by benzolium** [comment]
@andyyu2004 sure thing! Got it.

File: execution_introspection_test.go [hunk 1]
```diff
@@ -53,6 +53,9 @@ func TestIntrospectionQuery(t *testing.T) {
 
 	input CinemaInput {
 		name: String! = "SomeName"
+		cinema: CinemaInput = {name: "SomeOtherName"}
```
Comment (pkqk) [inline execution_introspection_test.go:56]: is this a valid input type? Would it create a infinite chain of `cinema` values.

Comment (benzolium) [reply execution_introspection_test.go:56]: Yes indeed:
https://spec.graphql.org/October2021/#sec-Input-Objects.Circular-References

**Pull request merged**